src/app/core/entity-components/entity-select/entity-select.module.ts

```diff
@@ -22,6 +23,7 @@ import { MatTooltipModule } from "@angular/material/tooltip";
     MatInputModule,
     ReactiveFormsModule,
     MatTooltipModule,
+    ViewModule,
```

Comment:
it seems you are not using anything from this here, only in the FormEntitySelectComponent of the other module?